js/src/main/scala/crystal/react/StateProvider.scala

```diff
@@ -1,28 +1,17 @@
 package crystal.react
 
 import crystal.ViewF
-import crystal.react.reuse.Reuse
+import crystal.react.hooks._
+import crystal.react.reuse._
 import japgolly.scalajs.react._
 import japgolly.scalajs.react.util.DefaultEffects.{ Sync => DefaultS }
 import japgolly.scalajs.react.vdom.html_<^._
-
-import implicits._
+import scala.reflect.ClassTag
```

Comment:
Is `ClassTag` gone in scala-3?

Reply:
It's still there. crystal has been using it and it compiles for Scala 3.